src/main/java/src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+package src.main.java;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected char taskType;
+
+    public Task(String description, char taskType) {
+        this.description = description;
+        this.isDone = false;
+        this.taskType = taskType;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String getWrittenIcon() { return (isDone ? "1" : "0"); }
```

Comment:
Perhaps this should [follow the coding standards](https://se-education.org/guides/conventions/java/basic.html#layout)?
```suggestion
    public String getWrittenIcon() { 
        return (isDone ? "1" : "0");
    }
```